crates/biome_js_analyze/src/analyzers/nursery/use_filenaming_convention.rs

```diff
@@ -217,23 +242,24 @@ pub(crate) enum FileNamingConventionState {
 pub struct FilenamingConventionOptions {
     /// If `false`, then consecutive uppercase are allowed in _camel_ and _pascal_ cases.
     /// This does not affect other [Case].
-    #[serde(
-        default = "default_strict_case",
-        skip_serializing_if = "is_default_strict_case"
-    )]
+    #[serde(default = "enabled", skip_serializing_if = "is_enabled")]
     pub strict_case: bool,
 
+    /// If `false`, then non-ASCII characters are allowed.
+    #[serde(default = "enabled", skip_serializing_if = "is_enabled")]
+    pub require_ascii: bool,
+
     /// Allowed cases for _TypeScript_ `enum` member names.
     #[serde(default, skip_serializing_if = "is_default_filename_cases")]
     pub filename_cases: FilenameCases,
 }
 
-const fn default_strict_case() -> bool {
+const fn enabled() -> bool {
```

Comment:
What's the purpose of this function?

Reply:
It is for serde serialization/deserialization. See line 249 and 245.